CONTRIBUTING.md

```diff
@@ -238,18 +240,19 @@ You can then consume packages from this package source, remember to check the [I
 
 ### Project Structure
 
-In `sdk\< Service Name >`, you will find projects for services that have already been implemented
+In `sdk\<ServiceName>`, you will find projects for services that have already been implemented.
 
 1. Client library projects needs to use the $(RequiredTargetFrameworks) *(defined in eng/Directory.Build.Data.props)* property in its TargetFramework while management library projects should use $(SdkTargetFx) _(defined in AzSdk.reference.props)_
-2. Projects of related packages are grouped together in a folder following the structure specified in [Repo Structure](https://github.com/Azure/azure-sdk/blob/master/docs/engineering-system/repo-structure.md)
-   - Client library packages are in a folder name like **_Microsoft.Azure.< ServiceName >_**
-   - Management library packages are in a folder named like **_Microsoft.Azure.Management.< Resource Provider Name >_**
-3. Each shipping package contains a project for their **generated** and /or **Customization** code
+2. Projects of related packages are grouped together in a folder following the structure specified in [Repo Structure](https://azure.github.io/azure-sdk/policies_repostructure.html)
+   - Client library packages are in a folder name like **_`Azure.<GroupName>.<ServiceName>`_**
+   - Management library packages are in a folder named like **_`Azure.Management.<ResourceProviderName>`_**
```

Comment:
Is this right? I thought it was Azure.ResourceManager.*

Reply:
Interesting. That would explain some things. I got the naming from: https://azure.github.io/azure-sdk/dotnet_introduction.html#general-azure-sdk-library-design
That shows `Azure.Management`, which I guess is out-of-date.

Reply:
Fixed. Found places in the document that are used for 'new' content that had `Azure.Management`, and changed it to use `Azure.ResourceManager`.